### PR TITLE
Adds issue management features

### PR DESCRIPTION
**Pull request by brendankowitz**
## Description
Adds issue management guidelines and github workflow

Based on: https://github.com/dotnet/aspnetcore/blob/master/docs/IssueManagementPolicies.md

### CODE REVIEW

**Comment by hansenms** [comment]
Could we add something here on channels for asking general FHIR questions not related to the FHIR Server for Azure. Specifically the kind of questions that should go to stackoverflow or the zulip chat. And possibly have a label for that. The questions/suggestions here should be for the FHIR server.

**Comment by brendankowitz** [comment]
I was more thinking this document is how issues are managed. Are you thinking something different to what we already have in https://github.com/microsoft/fhir-server#contributing?

**Comment by hansenms** [comment]
I was more thinking of calling out that there are better places to ask generic questions, like how to search for X, and so on than here.

**Comment by YazanMSFT** [comment]
How about putting a note with links to the appropriate place for questions not related to this server in the bug and user story templates?

**Comment by hansenms** [comment]
@YazanMSFT, yes, sounds like what I had in mind.

**Comment by brendankowitz** [comment]
@hansenms can you take another read through this?

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+labels: Question
+
+---
+
+**Question?**
+
+We are primarily tracking technical features, enhancements and bugs as Github issues.
+We will always accept your feedback here, but we ask that before posting your question, please see if it aligns with one of these categories:
+
+* Is your question answered in our documentation? - https://docs.microsoft.com/en-us/azure/healthcare-apis/fhir-oss-portal-quickstart)
```
Comment (hansenms) [inline .github/ISSUE_TEMPLATE/question.md]: link should be https://docs.microsoft.com/azure/healthcare-apis/

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+name: Question
+about: Open a question or discussion
+labels: Question
+
+---
+
+**Question?**
+
+We are primarily tracking technical features, enhancements and bugs as Github issues.
+We will always accept your feedback here, but we ask that before posting your question, please see if it aligns with one of these categories:
```
Comment (hansenms) [inline .github/ISSUE_TEMPLATE/question.md]: nit: We always appreciate your feedback, but we ask....

**Pull request closed without merging**